mmv1/templates/terraform/examples/eventarc-basic-tf.tf.erb

```diff
@@ -0,0 +1,116 @@
+# [START eventarc_terraform_enableapis]
```

Comment:
Sorry, forgot to mention earlier that we also have to rename this file to match with test name (`eventarc_basic_tf`).